singularity-eos/eos/eos_spiner.hpp

```diff
@@ -1889,7 +1907,9 @@ SpinerEOSDependsRhoSie::FillEos(Real &rho, Real &temp, Real &energy, Real &press
     }
   } else {
     lRho = toLog_(rho, lRhoOffset_);
-    if (!variadic_utils::is_nullptr(lambda)) lambda[0] = lRho;
+    if (!variadic_utils::is_nullptr(lambda)) {
+      IndexerUtils::Get<IndexableTypes::LogDensity>(lambda) = lRho;
```

Comment:
Include the `Lambda::lRho` argument explicitly for clarity, and similarly below.